inspire/modules/workflows/workflows/process_record_arxiv.py

```diff
@@ -110,7 +110,7 @@ class process_record_arxiv(RecordWorkflow):
                     with_author_keywords=True,
                 ),
                 filter_core_keywords(filter_kb="antihep"),
-                guess_coreness(),
+                #guess_coreness(),
```

Comment:
You can remove this change. I made it so that if there is no model prediction file, it will not fail. You'll still need to install the dependencies though as written in the README: https://github.com/inspirehep/inspire-next/blob/master/README.rst#core-prediction